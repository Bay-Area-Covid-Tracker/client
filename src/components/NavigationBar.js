import React from 'react';
import { Link } from 'react-router-dom';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import Button from '@material-ui/core/Button';
import { makeStyles } from '@material-ui/core/styles';

const flexGrow = 1
const useStyles = makeStyles(theme => ({
	root: { flexGrow, backgroundColor: '#ffffff', color: '#000000' },
	title: { flexGrow },
	links: { textDecoration: 'none', color: '#000000' }
}));

export default function NavigationBar() {
	const classes = useStyles();

	return (
		<AppBar className={classes.root} position="static">
			<Toolbar>
				<Typography variant="h6" className={classes.title}>
					Bay Area Covid-19 Tracker
				</Typography>
				<Link className={classes.links} to="/"><Button color="inherit">Home</Button></Link>
				<Link className={classes.links} to="/about"><Button color="inherit">About</Button></Link>
				<a href="https://github.com/Bay-Area-Covid-Tracker"
					rel="noopener noreferrer" 
					target="_blank" 
					className={classes.links}
				>
					<Button color="inherit">GitHub</Button>
				</a>
			</Toolbar>
		</AppBar>
	);
}
