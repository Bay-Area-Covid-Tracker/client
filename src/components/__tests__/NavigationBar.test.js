import React from 'react';
import { shallow } from 'enzyme';

import NavigationBar from '../NavigationBar';

describe('tests for Navigation', () => {
	describe('snapshot tests', () => {
		it('renders without crashing', () => {
			const tree = shallow(<NavigationBar />);
			expect(tree).toMatchSnapshot();
		});
	});
});
