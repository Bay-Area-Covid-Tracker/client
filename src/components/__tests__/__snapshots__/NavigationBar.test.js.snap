// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for Navigation snapshot tests renders without crashing 1`] = `ShallowWrapper {}`;
